fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/traci/VehicleAdd.java

```diff
@@ -35,7 +35,7 @@ public class VehicleAdd
      */
     @SuppressWarnings("WeakerAccess")
     public VehicleAdd() {
-        super(TraciVersion.HIGHEST);
+        super(TraciVersion.API_20);
```

Comment:
Why did this change? Was the API changed with the new version?

Reply:
Yes, `HIGHEST` is now `API_21` but this command should still work with older SUMO version, thus the change.